packages/suite/src/components/wallet/SearchAction.tsx

```diff
@@ -102,13 +109,9 @@ export const SearchAction = ({
                 size="small"
                 innerRef={inputRef}
                 innerAddon={
-                    <Icon
-                        name="search"
-                        cursorPointer
-                        size={16}
-                        color={theme.iconDefault}
-                        onClick={!isExpanded ? openAndSelect : undefined}
-                    />
+                    <SearchIconContainer onClick={!isExpanded ? openAndSelect : undefined}>
```

Comment:
Perhaps you don't need this at all if you add the onClick handler on the StyledInput itself.

Reply:
Not really because then it doesn't click on the icon :D